trlx/orchestrator/offline_orchestrator.py

```diff
@@ -1,76 +1,125 @@
+from typing import List, Union
+
+import numpy as np
 import torch
 
 from trlx.orchestrator import Orchestrator, register_orchestrator
 from trlx.pipeline.offline_pipeline import ILQLRolloutStorage
 
 
+def tokenize_dialogue(
```

Comment:
Why are we adding dialogue related functionality to the core api?

Reply:
I think having this func in trlx makes sense, it makes the completion split based preproc for ilql a bit clearer although we should probably pair it with a multi-turn example to show it working

Reply:
Perhaps including it is fine but I think it should be optional (and not the default)